docs/package-sign.md

```diff
@@ -84,213 +84,80 @@ Follow these steps to get the packaging and signing tool and set up the virtual
 
 mysql_odbc_sample.taco was created in e:\temp
 
-Enter keystore password:
-
-Enter password for alias test1year:(RETURN if same as keystore password)
-
-jar signed.
-
-Warning:
-
-No -tsa or -tsacert is provided and this jar is not timestamped. Without a timestamp, users may not be able to validate this jar after the signer certificate's expiration date (2020-10-06) or after any future revocation date.
-
 ```
 
+## Signing your packaged connector using jarsigner
 
-### Verify the connector with signature
-
-**Note:** You can use the package tool to package and sign the connector, and use the connector in Tableau Desktop 2019.x. Signature verification is supported starting in Tableau Desktop 2019.4 on Windows.  
-
-You can install and test your connector one of two ways:
-
--   Put your .taco file in the standard location, My Tableau Repository/Connectors, then launch Tableau Desktop. 
-
--   Put your .taco file anywhere you want. Launch Tableau Desktop with the packaged and signed connector with the option -DConnectPluginsPath . For example:
-
-    ```
-    "C:\Program Files\Tableau\Tableau 2019.4\bin\tableau.exe" -DConnectPluginsPath="d:\connector-plugin-sdk\connector-packager\packaged-connector" 
-    ```
-
-In both cases, when you launch Tableau Desktop, your connector is listed on the **Connect** pane, under **To a Server** > **More**.
+At this point, your connector has been packaged into a single `.taco` file. However, it will not be loaded automatically into Tableau unless you sign the file, or disable signature verification.
 
-![]({{ site.baseurl }}/assets/connect-pane.png) 
- 
-
-To connect, select the name of your connector, enter the information in the Connection dialog, and then click **Sign In**.
+### Why we require connectors to be signed
 
-![]({{ site.baseurl }}/assets/data-source-page.png)
+Connectors are sensitive parts of the Tableau code. They handle database authentication and communicate directly with your driver. By signing the connector, Tableau can verify the authenticity and integrity of the connector, and customers can be confident the the plugin author is who they say they are, and the `.taco` file itself has not been tampered with since it was signed.
 
-### (Optional) Verify the connector without a signature
+### Getting your connector signed
 
-You can verify a packaged connector without a signature. If everything is correct, other than the signature, the process should only skip the signature verification part and perform all the other steps correctly. This includes connecting to the data. To verify the connector without a signature, follow these steps:
+A packaged Tableau Connector (`.taco`) file is, functionally, a `.jar` file. Tableau checks that packaged connectors are signed by a trusted certificate authority before loading them, using the default java keystore in the JRE. Because a `.taco` file is fundamentally a `.jar` file, you can follow generic documentation for signing jar files.
```

Comment:
follow **generic** documentation for signing jar files. -> follow **Java** documentation for signing jar files.